autofit/non_linear/initializer.py

```diff
@@ -51,16 +50,16 @@ def from_config(cls, config):
                 lower_limit=ball_lower_limit, upper_limit=ball_upper_limit
             )
 
-    def initial_samples_from_model(self, total_points, model, fitness_function):
+    def initial_samples_from_model(self, total_points : int, model, fitness_function):

```

Comment:
Weird formatting. Probably best over new line.

The model should be type AbstractPriorModel